### DISCUSSION TITLE
Optimizing for Production

### DISCUSSION THREAD

**Discussion by deepaklth** [tailwindlabs/tailwindcss#13472] [Help]
I am using Tailwind in my angular project along with Angular material.
When I generate the angular build of my project, the size of styles.css file is around 150KB.
How can I reduce the  size of the style.css file.?
I have checked the style.css file has lots of redudant utlities class of tailwind.
Is there any way to remove those redudant classes from my style.css file at the time of generating angular build?

**Comment by wongjn** [comment]
You could consider:
- Making the [`content`](https://tailwindcss.com/docs/content-configuration) file globs as specific as possible with regards to source files actually used to render your UI. This can help reduce false-positive matches of class names that are not used.
- Using the [`blocklist`](https://tailwindcss.com/docs/content-configuration#discarding-classes) option to tell Tailwind not to generate CSS rules for the classes listed.
